grow/pods/pods.py

```diff
@@ -158,6 +163,18 @@ def _load_extensions(self, load_local_extensions=True):
             self._extensions_controller.register_extensions(
                 self.yaml.get('ext', []))
 
+    def _load_features(self):
+        config = self.yaml.get('features', {})
+        for key, value in config.iteritems():
+            # Features can be turned on with a True value.
+            # But the true does does not act as a configuration.
```

Comment:
Not sure I understand what this comment is saying, could you re-phrase?

Reply:
Also, can we make the legacy string placeholders use the same enable/disable as what you're creating here?

Reply:
Yeah, we can make the legacy placeholder use it too, I'll do that in a separate pull request though.

Reply:
I've also updated the phrasing here os it is hopefully easier to understand.